docs/install/build_from_source.md

```diff
@@ -203,68 +204,74 @@ It is recommended to set environment variable NCCL_LAUNCH_MODE to PARALLEL when
 
 ### Build MXNet with C++
 
-* To enable C++ package, just add `USE_CPP_PACKAGE=1` when you run `make` or `cmake`.
+* To enable C++ package, just add `USE_CPP_PACKAGE=1` when you run `make` or `cmake` (see examples).
 
 <hr>
 
 ### Usage Examples
 
-* `-j` runs multiple jobs against multi-core CPUs.
-
 For example, you can specify using all cores on Linux as follows:
 
 ```bash
-cmake -j$(nproc)
+cmake -GNinja .
+ninja -v
```

Comment:
Should we better recommend an out of source build?  
mkdir build && cd build
cmake -GNinja ..
ninja -v

Reply:
Ok